[2025-09-15T11:23:22.903+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_to_dds_enhanced.load_orders manual__2025-09-15T11:22:59.929655+00:00 [queued]>
[2025-09-15T11:23:22.936+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_to_dds_enhanced.load_orders manual__2025-09-15T11:22:59.929655+00:00 [queued]>
[2025-09-15T11:23:22.937+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 3
[2025-09-15T11:23:23.087+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_orders> on 2025-09-15 11:22:59.929655+00:00
[2025-09-15T11:23:23.173+0000] {standard_task_runner.py:57} INFO - Started process 1213 to run task
[2025-09-15T11:23:23.180+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'raw_to_dds_enhanced', 'load_orders', 'manual__2025-09-15T11:22:59.929655+00:00', '--job-id', '396', '--raw', '--subdir', 'DAGS_FOLDER/raw_to_dds.py', '--cfg-path', '/tmp/tmpqzscw7hg']
[2025-09-15T11:23:23.192+0000] {standard_task_runner.py:85} INFO - Job 396: Subtask load_orders
[2025-09-15T11:23:23.458+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-15T11:23:23.919+0000] {task_command.py:410} INFO - Running <TaskInstance: raw_to_dds_enhanced.load_orders manual__2025-09-15T11:22:59.929655+00:00 [running]> on host 3deaa5b4a23c
[2025-09-15T11:23:27.930+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_to_dds_enhanced' AIRFLOW_CTX_TASK_ID='load_orders' AIRFLOW_CTX_EXECUTION_DATE='2025-09-15T11:22:59.929655+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-15T11:22:59.929655+00:00'
[2025-09-15T11:23:31.731+0000] {python.py:183} INFO - Done. Returned value was: 50
[2025-09-15T11:23:31.899+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=raw_to_dds_enhanced, task_id=load_orders, execution_date=20250915T112259, start_date=20250915T112322, end_date=20250915T112331
[2025-09-15T11:23:31.902+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-09-15T11:23:32.447+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-09-15T11:23:32.508+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
